pkgs/applications/video/handbrake/default.nix

```diff
@@ -84,6 +92,11 @@ _EOF
     sed -e 's/^[[:space:]]*\(meson\|ninja\|nasm\)[[:space:]]*= ToolProbe.*$//g' \
         -e '/    ## Additional library and tool checks/,/    ## MinGW specific library and tool checks/d' \
         -i make/configure.py
+
+    # Prevent the configure script from failing on Darwin if xcodebuild isn't
+    # available, which it isn't in the Nix context. (The actual build goes fine
+    # without xcodebuild.)
+    sed -e '/xcodebuild = ToolProbe/s/abort=.\+)/abort=False)/' -i make/configure.py
```

Comment:
I will open a bug report with upstream when I have a chance.

Reply:
It’s a little late but I opened https://github.com/HandBrake/HandBrake/issues/3048.

Reply:
It is way better than never.

We can not permit the cost to gather all types of upstream patching, FP Nixpkgs is upstream Report generator, or otherwise, Nixpkgs would become just one huge patch of the IT world.